bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/inverter.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fronius"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Powerinverter Thing Type -->
+	<thing-type id="powerinverter">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Fronius Symo Inverter</label>
+		<description>Fronius Symo Power Inverter</description>
+
+		<channels>
+			<channel id="inverterdatachannelpac" typeId="inverter_pac"/>
```

Comment:
You renamed all channel types, this is of course a breaking change (for users using UI to define things).
I mean you renamed `pac` into `inverter_pac`. Is it really necessary ?
I would suggest, if possible, to not rename any channel type name (and channel name).

Reply:
I didn't know that changing channel type id would break anything? I thought only channel id matters?

Reply:
Yes, it is (only) a problem when the thing was created with UI.

Reply:
@lolodomo are you sure that changing typeId is a user breaking change?

Reply:
See #12299

Reply:
> See #12299

That was the change of the `channel id` that caused the problem, not `channel-type id`.
My understanding is that the change in `channel-type id` has no bearing in the referenced issue whatsoever. My understanding is that you could even call it any random character and it would not matter.  

In any case I'll find some time to test this.

I would really rather rename the `channel-type id` (not `channel id`) to tidy things up.

Reply:
I did some tests and looked into the json database, and it appears that you're correct and I was mistaken. A change in the channel type id matters for things created with UI as you said. I will revert my changes.